CMakeLists.txt

```diff
@@ -1,18 +1,32 @@
 cmake_minimum_required(VERSION 3.5.1)
-project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
-
-include(ExternalProject) # For ExternalProject_Add command
 
 # This is where we will install artifacts during the build stage
-set(stage_directory "${CMAKE_BINARY_DIR}/stage/${CMAKE_INSTALL_PREFIX}")
+# TODO: change stage_dir to CMAKE_INSTALL_DIR
+set(stage_directory "${CMAKE_BINARY_DIR}/stage")
+
+set(CMAKE_BUILD_TYPE_INIT "Release")
+
+project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
```

Comment:
This should be the second line in file, it actually initializes a bunch of variables.